src/client.ts

```diff
@@ -138,6 +138,34 @@ interface ClientArgs {
   timeout?: TimeDelta;
 }
 
+interface ResultWithEvaluations {
+  id: string;
+  runId: string;
+  hash: string;
+  datasetItemId: string;
+  durationMs: number;
+  events?: Event[];
+  body: {
+    id: string;
+    name: string;
+    value: string;
+    contentType: HumanReviewFieldContentType;
+  }[];
+  output: {
+    id: string;
+    name: string;
+    value: string;
+    contentType: HumanReviewFieldContentType;
+  }[];
```

Comment:
we should make these as unknown and let folks pass in types for them using generics in the get results method